components/atom/label/README.md

```diff
@@ -0,0 +1,23 @@
+# AtomLabel
+
+> Description
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-atom-label --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import AtomLabel from '@s-ui/react-atom-label'
+
+return (<AtomLabel />)
+```
+
+
+> **Find full description and more examples in the [demo page](#).**
```

Comment:
please the future link to the demo page `https://sui-components.now.sh/workbench/atom/label/demo`